packages/at_secondary_server/lib/src/enroll/enroll_datastore_value.dart

```diff
@@ -25,6 +25,13 @@ class EnrollDataStoreValue {
       _$EnrollDataStoreValueFromJson(json);
 
   Map<String, dynamic> toJson() => _$EnrollDataStoreValueToJson(this);
+
+  Map<String, dynamic> toJsonExtended() {
+    final m = toJson();
+    m['status'] = approval?.state;
+    m['namespace'] = m['namespaces'];
+    return m;
+  }
```

Comment:
the handling of enrollments JSON was a bit wack ... this helped clean it up a little